AmazonChimeSDKDemo/AmazonChimeSDKDemo/JoiningViewController.swift

```diff
@@ -18,8 +18,11 @@ class JoiningViewController: UIViewController, UITextFieldDelegate {
     @IBOutlet var joinWithoutCallKitButton: UIButton!
     @IBOutlet var joinAsIncomingCallButton: UIButton!
     @IBOutlet var joinAsOutgoingCallButton: UIButton!
+    @IBOutlet var testSettingsButton: UIButton!
 
     private let toastDisplayDuration = 2.0
+    private let meetingPresenter = MeetingPresenter.shared()
+    var model: TestSettingsModel?
```

Comment:
if we expect this model to always be there, it does not have to be a optional var. It can be a let with inline init, rather than in viewDidLoad